app/client/src/components/shared/AddDataWidget.SearchPanel.tsx

```diff
@@ -740,59 +790,60 @@ function ResultCard({ result }: ResultCardProps) {
   /**
    * Adds non-tots layers as reference portal layers.
    */
-  function addRefLayer() {
+  const addRefLayer = useCallback(async () => {
```

Comment:
```suggestion
  const addRefLayer = useCallback(() => {
```

I don't really like the idea of using async inside of a useCallback like this. Was there a specific need for this? 

The next 3 items relate to reverting this back to the way it was while still making TypeScript happy.

Reply:
No particular need, I'll revert that back.